Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -948,48 +997,113 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Gin
                                 }
                             }
                         }
-                        if (string.IsNullOrEmpty(runsetOperationConfigMail.MailSettings.MailFrom) == false)
+                        if (runsetOperationConfigMail.MailSettings.MailFrom != null)
                         {
                             mailOperation.MailFrom = runsetOperationConfigMail.MailSettings.MailFrom;
                             mailOperation.Email.MailFrom = runsetOperationConfigMail.MailSettings.MailFrom;
                         }
-                        if (string.IsNullOrEmpty(runsetOperationConfigMail.MailSettings.MailTo) == false)
+                        if (runsetOperationConfigMail.MailSettings.MailTo != null)
                         {
                             mailOperation.MailTo = runsetOperationConfigMail.MailSettings.MailTo;
                             mailOperation.Email.MailTo = runsetOperationConfigMail.MailSettings.MailTo;
                         }
-                        if (string.IsNullOrEmpty(runsetOperationConfigMail.MailSettings.MailCC) == false)
+                        if (runsetOperationConfigMail.MailSettings.MailCC != null)
                         {
                             mailOperation.MailCC = runsetOperationConfigMail.MailSettings.MailCC;
                         }
-                        if (string.IsNullOrEmpty(runsetOperationConfigMail.MailSettings.Subject) == false)
+                        if (runsetOperationConfigMail.MailSettings.Subject != null)
                         {
                             mailOperation.Subject = runsetOperationConfigMail.MailSettings.Subject;
                             mailOperation.Email.Subject = runsetOperationConfigMail.MailSettings.Subject;
                         }
 
-                        if (string.IsNullOrEmpty(runsetOperationConfigMail.Comments) == false)
+                        //report mail content
+                        if (runsetOperationConfigMail.Comments != null)
                         {
                             mailOperation.Comments = runsetOperationConfigMail.Comments;
                         }
+                        if (runsetOperationConfigMail.BodyContentType != null)
+                        {
+                            switch (runsetOperationConfigMail.BodyContentType)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace this 'switch' statement with 'if' statements to increase readability.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5700269)